be/src/io/fs/local_file_reader.h

```diff
@@ -26,6 +26,20 @@
 #include "io/fs/path.h"
 #include "util/slice.h"
 
+namespace doris {
+
+struct StorePath;
+struct DataDirInfo;
+
+// data dir in be config, currently include storage dir and spill disk dir
+extern std::vector<DataDirInfo> be_config_data_dir_list;
```

Comment:
avoid using global variable because it is related with compile component .
Move it to LocalFileReader as a static variable.